src/utils/url.ts

```diff
@@ -0,0 +1,14 @@
+import { URL_PARAMS, PATHS } from "enums";
+
+export const setURLParam = (param: URL_PARAMS, paramValue: string, path: PATHS) => {

```

Comment:
первый и второй аргумент можно передать как объект
```suggestion
export const setURLParam = ({ param, value }: URLParam, path: PATHS) => {

```